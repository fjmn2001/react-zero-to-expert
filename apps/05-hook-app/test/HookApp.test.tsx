import { describe, expect, test } from "vitest"
import { render } from "@testing-library/react"
import HookApp from "../src/HookApp"

describe("Testing HookApp.tsx", () => {
  test("it should show correctly", () => {
    const { container } = render(<HookApp />)
    expect(container).toMatchSnapshot()
  })
})
