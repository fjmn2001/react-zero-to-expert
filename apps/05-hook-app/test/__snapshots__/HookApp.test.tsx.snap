// Vitest Snapshot v1

exports[`Testing HookApp.tsx > it should show correctly 1`] = `
<div>
  <h1>
    Hi there!
  </h1>
</div>
`;
